sonar-db/src/main/java/org/sonar/db/permission/PermissionRepository.java

```diff
@@ -133,19 +87,25 @@ public void applyPermissionTemplate(DbSession session, String templateUuid, Comp
   }
 
   private void applyPermissionTemplate(DbSession session, String templateUuid, ComponentDto project, @Nullable Long currentUserId) {
-    PermissionTemplate permissionTemplate = dbClient.permissionTemplateDao().selectPermissionTemplateWithPermissions(session, templateUuid);
+    PermissionTemplate template = dbClient.permissionTemplateDao().selectPermissionTemplateWithPermissions(session, templateUuid);
     updateProjectAuthorizationDate(session, project.getId());
     dbClient.roleDao().deleteGroupRolesByResourceId(session, project.getId());
     dbClient.userPermissionDao().delete(session, null, project.uuid(), null);
 
-    List<PermissionTemplateUserDto> usersPermissions = permissionTemplate.getUserPermissions();
-    usersPermissions.forEach(userPermission -> insertUserPermission(project.getId(), userPermission.getUserId(), userPermission.getPermission(), false, session));
+    List<PermissionTemplateUserDto> usersPermissions = template.getUserPermissions();
+    // FIXME String organizationUuid = template.getTemplate().getOrganizationUuid();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)